tests/storage/export.go

```diff
@@ -1284,7 +1255,10 @@ var _ = SIGDescribe("Export", func() {
 			vm.Spec.RunStrategy = virtpointer.P(v1.RunStrategyAlways)
 		}
 		vm = createVM(vm)
-		stopVM(vm)
+		if libstorage.IsStorageClassBindingModeWaitForFirstConsumer(sc) {
+			err = virtClient.VirtualMachine(vm.Namespace).Stop(context.Background(), vm.Name, &v1.StopOptions{})
+			Expect(err).ToNot(HaveOccurred())
+		}
```

Comment:
AFAIU from the existing logic, this condition is new.
Could you please add reasoning in the commit message explaining why was it added?

Reply:
sure, added

Reply:
Could you please elaborate how "avoid stopping when the VM wasn't started" relates to the newly added condition (which AFAIU relates to a storageClass)?

Reply:
The same condition is used to decide whether we start the VM a few lines above

Reply:
I see, thank you for the explanation.
Please consider adding in follow-up, a flag to unify the two conditions.